blog/12-05-2023-build-a-document-tamper-proof-verification-smart-contract-dapp-on-celo-blockchain/build-a-document-tamper-proof-verification-smart-contract-dapp-on-celo-blockchain.md

```diff
@@ -0,0 +1,844 @@
+---
+title: Build a document tamper proof verification smart contract dApp on Celo blockchain 
+description: This tutorial will guide you through building a decentralized application on the Celo blockchain that verifies the integrity of documents.
+authors:
+  - name: Jovan Mwesigwa
+    title: Software Engineer
+    url: https://github.com/JovanMwesigwa
+    tags: [celo, celosage, intermediate, 'solidity', 'metamask', 'nodejs', 'javascript']
+    hide_table_of_contents: true
+    slug: /tutorials/build-a-document-tamper-proof-verification-smart-contract-dapp-on-celo-blockchain
```

Comment:
please remove this tabs, "tags", "hide_table_of_contents" and "slug" doesn't make part of "authors" section